### PR TITLE
feat(deps): bump to Orchestrate v2.4.0-rc2

### PR DESCRIPTION
**Pull request by julien-marchand**

### CODE REVIEW

File: files/orchestrate/package.json [hunk 1]
```diff
@@ -66,13 +66,13 @@
...
     "dotenv-cli": "~3.1.0",
-    "ethers": "~4.0.47",
+    "ethers": "^5.0.15",
     "husky": "~4.2.5",
     "lint-staged": "~10.2.2",
     "pegasys-orchestrate": "~3.1.1",
     "prettier": "~2.0.5",
     "rimraf": "~3.0.2",
-    "truffle": "~5.1.24",
+    "truffle": "^5.1.47",
```
Comment (Kriys94) [inline files/orchestrate/package.json:75]: Can we keep ~ instead of ^

File: files/orchestrate/README.md [hunk 2]
```diff
@@ -7,22 +7,22 @@
...
-                  
+
 
 ## Prerequisites
 
 To run these tutorials, you must have the following installed:
 
 - [Docker and Docker-compose](https://docs.docker.com/compose/install/)
-
+f
```
Comment (benjamincburns) [inline files/orchestrate/README.md:17]: typo?

File: files/orchestrate/README.md [hunk 3]
```diff
@@ -7,22 +7,22 @@
...
 
 
-| ⚠️ **Note**: This has only been tested on Windows 10 Build 18362 and Docker >= 17.12.2                                                                                                                                                                                                                                                                                              |
-| ---                                                                                                                                                                                                                                                                                                                                                                                |
+| ⚠️ **Note**: This has only been tested on Windows 10 Build 18362 and Docker >= 17.12.2
+| ---
 
-- On Windows ensure that the drive that this repo is cloned onto is a "Shared Drive" with Docker Desktop
+- On Windows ensure that the drive, that this repo is cloned onto, is a "Shared Drive" with Docker Desktop
 - On Windows we recommend running all commands from GitBash
```
Comment (benjamincburns) [inline files/orchestrate/README.md:26]: I think this recommendation should change to WSL2 w/ Docker Desktop installed such that it uses the WSL2 execution engine. I use it for testing all of this stuff, and it's a lot more reliable and easier to set up than other alternatives.

Comment (benjamincburns) [reply files/orchestrate/README.md:26]: Also that'll bring this README in line with the rest of the quickstart

File: files/orchestrate/src/consume/consume.ts [hunk 4]
```diff
@@ -17,7 +18,7 @@ export const consume = async () => {
       console.error('Transaction failed with error: ', value.errors)
     } else {
       console.log('RequestId:', value.id)
-      console.log('Receipt: ', value.receipt)
+      console.log('Receipt: ', util.inspect(value.receipt, false, null, true))
```
Comment (benjamincburns) [inline files/orchestrate/src/consume/consume.ts:21]: I'd advise using `JSON.stringify(value.receipt, null, 2)` here to pretty print the receipt as a JSON object w/ an indent level of 2. `util.inspect` can truncate sometimes.

File: files/orchestrate/README.md (hunk 2 above)
Comment (julien-marchand) [reply files/orchestrate/README.md]: Yes... thanks!

File: files/orchestrate/src/consume/consume.ts (hunk 4 above)
Comment (julien-marchand) [reply files/orchestrate/src/consume/consume.ts]: Thanks, updated

File: files/orchestrate/README.md (hunk 3 above)
Comment (julien-marchand) [reply files/orchestrate/README.md:26]: We aren't Windows users so happy to go with your suggestion. Let's treat that in another MR?

**Review by benjamincburns — APPROVED** [review]

**Pull request merged**